java/pom.xml

```diff
@@ -612,6 +612,13 @@
       <artifactId>mockito-junit-jupiter</artifactId>
       <version>2.25.1</version>
       <scope>test</scope>
+      <!-- Mockito  contains old Hamcrest classes, causes issues with JUNIT -->
+      <exclusions>
+        <exclusion>
+          <groupId>org.hamcrest</groupId>
+          <artifactId>hamcrest-core</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
Why is this added ? What errors did you get to require this addition ?

Reply:
This was in the commit history and I wasn't sure if it should be removed, i have removed it here https://github.com/sirensolutions/arrow/pull/16/commits/5d79a1902316072bb4b05250736b1fe669182141